pom.xml

```diff
@@ -79,6 +79,11 @@
 			<artifactId>h2</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>com.googlecode.json-simple</groupId>
+			<artifactId>json-simple</artifactId>
+			<version>1.1.1</version>
+		</dependency>
 	</dependencies>
 
```

Comment:
Can I update this before jparest is merged, or do I need to wait for the other PR to go in first?

Reply:
I think you can update the version here now whilst the PR is still open.
But yeah it will have to wait for the JpaRest one to go through first before this one can merge.